app/views/updateUser.ejs

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>
+      <%=title %>
+  </title>
+</head>
+<body>
+  <% for(var i in messages) { %>
+    <div class="flash"><%=messages[i] %></div>
+  <% } %>
+  <form action="/updateUser" method="post">
+    <div>
+      <label>First Name:</label>
+      <input type="text" name="firstName" value="<%=user.firstName %>" />
+    </div>
+    <div>
+      <label>Last Name:</label>
+      <input type="text" name="lastName" value="<%=user.lastName %>" />
+    </div>
+    <div>
+      <label>Phone:</label>
+      <input type="text" name="phone" value="<%=user.phone %>" />
+    </div>
+    <div>
+      <label>Email:</label>
+      <input type="text" name="username" value="<%=user.username %>" />
+    </div>
+    <div>
+      <label>Password:</label>
```

Comment:
Maybe we need another field for repeating the password

Reply:
@rivendell10 Do you think we should mess with the added password authentication here and in the signup view now? Or merge these changes in and add issues for the other user authentication pieces to work on later (the lost password email, password add/change verification, etc.)?

Reply:
Ok, you are right. Befor we spend too much time on these details it is better to focus on more important components of the application.